README.md

```diff
@@ -437,6 +437,31 @@ iban = Ibandit::IBAN.new(
 iban.iban                     # => "GB60BARC20000055779911"
 ```
 
+### Pseudo-IBANs
+
+In some countries, it is not possible to recover local banking details from an
+IBAN.  For this reason, Ibandit has a concept of a *pseudo-IBAN*.  This is a
+string with a similar structure to an IBAN that can be decomposed into local
+banking details.  Pseudo-IBANs can be recognized by the fact that they have `ZZ`
+as the third and fourth characters (these would be check digits for a regular
+IBAN).
+
+```
+iban = Ibandit::IBAN.new(
+  country_code: 'SE',
+  branch_code: '7507',
+  account_number: '1211203'
+)
+iban.pseudo_iban              # => "SEZZZ7507ZZZ1211203"
```

Comment:
Yeah, that might be a little clearer? Not mega-important though, I don't think we really need to optimise for readability too much.